{
  Throwable cause=e.getCause();
  logger.warn("exceptionCaught() UnexpectedError happened. state:{} Caused:{}",state.getString(),cause.getMessage(),cause);
}
